packages/synapse-react-client/src/components/widgets/query-filter/FacetFilterControls.tsx

```diff
@@ -109,23 +119,6 @@ export function applyChangesToValuesColumn(
   onChangeFn(result)
 }
 
-//rangeChanges
-export const applyChangesToRangeColumn = (
-  lastRequest: QueryBundleRequest | undefined,
-  facet: FacetColumnResultRange,
-  onChangeFn: (result: FacetColumnRequest[]) => void,
-  values: string[],
-) => {
-  facet.columnMin = values[0]
-  facet.columnMax = values[1]
-  const changedFacet = convertFacetColumnRangeRequest(facet)
-  const result = patchRequestFacets(
-    changedFacet,
-    lastRequest?.query?.selectedFacets,
-  )
-  onChangeFn(result)
-}
```

Comment:
Now handled by `useImmutableTableQuery` and the callback is provided by the QueryContext